### PR TITLE
Added additional resources for frontend developers

### PR DESCRIPTION
**Pull request by stallardjw**

### CODE REVIEW

File: src/assets/data/resources.json [hunk 1]
```diff
@@ -65,4 +65,22 @@
             "design"
         ]
     }
```
Comment (TenTraicion) [inline src/assets/data/resources.json:67]:
```suggestion
    },
```

File: src/assets/data/resources.json [hunk 2]
```diff
@@ -65,4 +65,22 @@
...
+    {
+        "title": "Brackets",
+        "description": "Brackets is a modern text editor that makes it easy to design in the browser. It's crafted from the ground up for web designers and front-end developers.",
+        "url": "https://brackets.io",
+        "tags": [
+            "editor",
+            "developer"
+        ]
+    }
```
Comment (TenTraicion) [inline src/assets/data/resources.json:76]:
```suggestion
    },
```

File: src/assets/data/resources.json [hunk 3]
```diff
@@ -65,4 +65,22 @@
...
+    {
+        "title": "Bootstrap",
+        "description": "Powerful, extensible, and feature-packed frontend toolkit. Build and customize with Sass, utilize prebuilt grid system and components, and bring projects to life with powerful JavaScript plugins.",
+        "url": "https://getbootstrap.com",
+        "tags": [
+            "tools",
+            "developer"
+        ]
+    }
```
Comment (TenTraicion) [inline src/assets/data/resources.json:85]:
```suggestion
    },
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@stallardjw, There should be a comma "," after the end of every resource. Kindly do the recommended changes and commit. Thanks!

**Review by TenTraicion — APPROVED** [review]

**Comment by clebsonf** [comment]
@stallardjw 
thanks for the contribution
Can you also help by giving a star on the project or an upvote on the product hunter? Of course, if you feel like it

Product hunter:
https://www.producthunt.com/posts/octobear-hub

**Pull request merged**